### DISCUSSION TITLE
The publish button on GitHub Desktop

### DISCUSSION THREAD

**Discussion by SHADOWELITE7** [community/community#22914] [New to GitHub]
<p>I have looked at similar posts regardings to this topic but mine isn’t about adding an existing repo. Here is my question:</p>
<p>If I made a repo and want to push the publish button, will it give me options to put it to my profile or Organization?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/070781683cd463fc1671b766b520ca18ee5094d6.png" data-download-href="/uploads/short-url/10bqd2aLKDt4okTzhSHGhCV6IlM.png?dl=1" title="Screen Shot 2018-07-07 at 10.11.19 AM.png"><img src="https://user-images.githubusercontent.com/1038121/181073697-de809766-6be0-43c7-9ca6-b04cf8a633af.png" alt="Screen Shot 2018-07-07 at 10.11.19 AM.png" data-base62-sha1="10bqd2aLKDt4okTzhSHGhCV6IlM" width="665" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/070781683cd463fc1671b766b520ca18ee5094d6_2_665x499.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/070781683cd463fc1671b766b520ca18ee5094d6_2_997x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/070781683cd463fc1671b766b520ca18ee5094d6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/070781683cd463fc1671b766b520ca18ee5094d6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-07-07 at 10.11.19 AM.png</span><span class="informations">1116×838 42.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Selecting “None” will add the repo to your personal account. Selecting an organization will add it to that organization.</p>

**Comment by SHADOWELITE7** [comment]
<p>Thank you,<br>
Also, could the none be changed to profile in some other new update.</p>

**Comment by lee-dohm** [comment]
<p>Feel free to submit an issue at <a href="https://github.com/desktop/desktop/issues/new" rel="nofollow noopener">https://github.com/desktop/desktop/issues/new</a> :+1:</p>
